worlds/kh1/Rules.py

```diff
@@ -0,0 +1,553 @@
+from BaseClasses import CollectionState, MultiWorld, LocationProgressType
+from .Locations import get_locations_by_category
```

Comment:
```suggestion
from BaseClasses import CollectionState, MultiWorld
```
Recommended that you remove unused imports